packages/odyssey-react-mui/src/labs/TopNav/UserProfile.tsx

```diff
@@ -37,6 +38,7 @@ const UserProfileIconContainer = styled("div", {
 const UserProfileInfoContainer = styled("div")(() => ({
   display: "flex",
   flexDirection: "column",
+  textAlign: "left",
```

Comment:
Doesn't really seem like we need any of these styles. Fairly certain we can remove this styled component entirely and render either a plain `div` or a fragment

Reply:
The `textAlign` is definitely needed (the default styles for `<button>` have `text-align: center`). As for the rest 🤷 ; the UserProfile component itself needs work more generally, I was just making modifications where necessary to get this Popover to work.

Reply:
No worries. Just figured I'd flag it. If you need the `text-align` for a button, I'd probably just pull that button into it's own styled component and apply this there. That way it's a bit more clear what is happening/why this declaration is here.

Reply:
I think I wasn't very clear in my description. 

In UserProfilePopover, I was wrapping this component with a MuiButton ([here](https://github.com/okta/odyssey/pull/2406/files#diff-4247df56544a88b974863fbda959827f387802aed697c828f00ac668343122b2R98)). In that scenario, a `text-align` from the button is causing a different layout here in the UserProfile. So this is to reset that / enforce the intended design of this component.

Reply:
Thanks for the extra context, I seen where you were wrapping this in a button. I was trying to say we should style that^ button with `textAlign: "left"` directly, rather than applying it here. The reasoning for the text align declaration would be more clear in that scenario, I think

Reply:
I think my point, though, is that the UserProfile component should always be left aligned regardless of any container it is put in. In that case, it should ensure that it always has the correct alignment no matter what, which is why I'd advocate for having it here at a minimum. Basically, I'm suggesting we insulate this from any consumer doing something silly since this component is exported from Labs by itself as well.

tl;dr: what if someone did `<center><UserProfile /></center>`? We'd want the UserProfile component to render with proper alignment no matter what.